elide-core/src/main/java/com/yahoo/elide/core/PersistentResource.java

```diff
@@ -702,11 +704,17 @@ public PersistentResource getRelation(String relation, String id) {
      * @return collection relation
      */
     public Set<PersistentResource> getRelation(String relationName) {
-        if (requestScope.getTransaction() != null && !requestScope.getPredicates().isEmpty()) {
-            final Class<?> entityClass = dictionary.getParameterizedType(obj, relationName);
-            final String valType = dictionary.getBinding(entityClass);
-            final Set<Predicate> filters = new HashSet<>(requestScope.getPredicatesOfType(valType));
-            return getRelation(relationName, filters);
+        // if pagination, then sorting with filters...
+        boolean hasFilters = !requestScope.getPredicates().isEmpty();
```

Comment:
we need to check if there are filters for this particular type.  If you look at the elide docs, elide filters are a bit more expressive and allow filtering things like includes as well as the collection.